agave_app/agaveGui.cpp

```diff
@@ -483,6 +481,9 @@ agaveGui::onRenderAction()
 
   // copy of camera
   CCamera camera = m_glView->getCamera();
+  // extract ViewerWindow from GLView3D to hand to RenderDialog
+  ViewerWindow* renderer = m_glView->borrowRenderer();

```

Comment:
What did this change to `ViewerWindow` from `IRenderWindow` do?

Reply:
The idea here was that the RenderDialog might need to use more things from the GLView than just its Renderer.  
It used to be GLView-->(has a ) IRenderWindow,  but now GLView has-a ViewerWindow has-a IRenderWindow.

Reply:
There may soon be a Great Renaming of these windows and renderer things